stories/f-select.stories.mdx

```diff
@@ -642,19 +642,22 @@ Use to provide a custom template in option list
 	<Story name="option-template">
 		{args => {
 			const [value, setValue] = useState(null);
-			const optionTemplate = function (option) {
-				return html`<f-div
-					><f-div direction="column" gap="x-small"
-						><f-text>${option.title}</f-text
-						><f-text ellipsis>${option.data.name}, ${option.data.email}</f-text></f-div
-					></f-div
-				>`;
+			const optionTemplate = function (option, isSelected) {
+				return !isSelected
```

Comment:
ternary operators are good for small conditions,
Can you please convert this to `if else` blocks, it will increase code readability